pyof/v0x01/common/action.py

```diff
@@ -213,47 +220,50 @@ def __init__(self, vlan_pcp=None):
 class ActionDLAddr(ActionHeader):
     """Action structure for :attr:`ActionType.OFPAT_SET_DL_SRC` or _DST."""
 
-    dl_addr_type = UBInt16(enum_ref=ActionType)
+    action_type = UBInt16(enum_ref=ActionType)
```

Comment:
This looks like is better to be on `__init__` of class. A attribute action_type, when passed the init just fill the action_type value, that is already defined on `ActionHeader` class.